weed/storage/store_ec_interval_read_test.go

```
@@ -86,6 +87,27 @@ func TestReadEcShardNeedleSpanningBlocks(t *testing.T) {
 	}
 	ecVolume.ShardLocationsRefreshTime = time.Now()
 	ecVolume.ShardLocationsLock.Unlock()
+	return ecVolume
+}
+
+func randomNeedleOfSize(id uint64, size int) *needle.Needle {
+	n := new(needle.Needle)
+	n.Id = types.Uint64ToNeedleId(id)
+	n.Data = make([]byte, size)
+	rand.Read(n.Data)
```

golangci-lint (2.12.2)

[error] SA1019: rand.Read has been deprecated since Go 1.20 because it shouldn't be used: For almost all use cases, [crypto/rand.Read] is more appropriate. If a deterministic source is required, use [math/rand/v2.ChaCha8.Read]. 

(staticcheck)